wiki/How_You_Can_Help!/en.md

```diff
@@ -1,80 +1,78 @@
+## Help answer technical, beatmap, and gameplay support
 
-We have moderators doing their best to keep chat under control, but there are 25 channels (and counting) and even more forums, making it hard for us to cover all activity every minute of the day. Fortunately, you can help us very easily!
+Lend someone a help when they encounter an issue with the game! Whether its a driver issue, a beatmap question, or gameplay questions, you can answer it if you spend enough time with osu!
 
-### Where to start:
+It doesn't matter if you don't know well enough of the game, you can always follow along and learn from reading, so when the next occurence of the same problem appears in the future, you can answer it as well.
 
--   Follow your favourite channels on in-game chat, and forums on the web.
--   If you see something happening which disturbs you (or would generally be found inappropriate), use the `!report` command to notify moderators.
--   You can use the `!report` command for anything that happens anywhere in osu!. Your reports will always be read.
+You can begin with lending people a hand on these following mediums:
 
-## Help answer technical, beatmap and gameplay support questions!
+- Choose a forum that sparks your interest:
+   - [Technical Support](/community/forums/5)
+   - [Beatmapping Support](/community/forums/56)
+   - [Storyboarding](/community/forums/20)
+   - [Gamplay and Rankings](/community/forums/13)
 
-Every day I try to take care of any outstanding tech support questions, but a lot of these can be answered by anyone who has spend time around here and knows how things generally work. Even if you don't think you know stuff well enough, you can follow along, read and learn, then start helping at a later point!
+- Mark the forum as read using the link at the top right.
+- Follow any new threads, reading each one and helping out when you can. You can use the unread icons to make sure you keep up.
+- Read the [available guides](/wiki/List_of_Guides "List of Guides") to polish up your osu! knowledge. If the guides in there doesn't satisfy you, you can contribute to them!
 
-### Where to start:
+## Help check/mod beatmaps!
 
--   Choose a forum: [Technical Support](/community/forums/5), [Beatmapping Help](/community/forums/56), [Storyboarding](/community/forums/20) or [Gameplay and Rankings](/community/forums/13) based on what you find most interesting.
--   Mark the forum as read using the link at the top-right.
--   Follow any new threads, reading each one and helping out where you can. You can use the unread icons to make sure you keep up!
--   Read the [available guides](/wiki/List_of_Guides "List of Guides") and polish up on your osu! knowledge. Hell, contribute to the guides if you can!
+osu! has a dedicated team of reviewers called "modders" to check if the beatmaps are ready to be ranked and played by everyone. However, that doesn't stop you from contributing in this sector, so here's a few things you can do to help:
```

Comment:
I wouldn't really call modders a team.. I think what the original meant here are the official teams (BNs, QAT)